Sources/AppStoreConnectClient/AppStoreConnectClient.swift

```diff
@@ -60,40 +60,92 @@ public struct AppStoreConnectClient {
                 return json.data.compactMap({ Application(schema: $0) })
             }
         case .badRequest(let result):
-            throw AppStoreConnectError.badRequest
+            switch result.body {
+            case .json(let json):
+                throw AppStoreConnectError.badRequest(errors: handleError(from: json))
+            }
         case .forbidden(let result):
-            throw AppStoreConnectError.forbidden
+            switch result.body {
+            case .json(let json):
+                throw AppStoreConnectError.forbidden(errors: handleError(from: json))
+            }
         case .unauthorized(let result):
-            throw AppStoreConnectError.unauthorized
+            switch result.body {
+            case .json(let json):
+                throw AppStoreConnectError.unauthorized(errors: handleError(from: json))
+            }
         case .undocumented(let statusCode, _):
             throw AppStoreConnectError.serverError(errorCode: statusCode)
         }
     }
     
+    /// Converts an error response into a formatted string.
+    /// - Parameter response: The error response object.
+    /// - Returns: A formatted string describing the error response, or `nil` if the response is empty.
+    private func handleError(from response: Components.Schemas.ErrorResponse) -> String? {
```

Comment:
Move it to error implementation

Reply:
Need to correct the name of this function - it should be something like "errorString(from..." or "errorDescription(from..."

Reply:
moved to AppStoreConnectError.swift